src/bin/cratesfyi.rs

```diff
@@ -584,7 +585,9 @@ impl Context {
     fn storage(&self) -> Result<Arc<Storage>, Error> {
         Ok(self
             .storage
-            .get_or_try_init::<_, Error>(|| Ok(Arc::new(Storage::new(self.pool()?))))?
+            .get_or_try_init::<_, Error>(|| {
+                Ok(Arc::new(Storage::new(self.pool()?, &*self.config()?)?))
+            })?
             .clone())
     }
```

Comment:
It's unrelated to your PR, but we should be using `once_cell::sync::Lazy`s inside of `Context` instead of `OnceCell`s

Reply:
We can't use `Lazy` (or it's impractical anyway). Those `OnceCell`s depend on each other, but `Lazy` requires the initializer closure to be defined when it's instantiated.